objects.py

```diff
@@ -263,25 +274,14 @@ def power(self) -> float:
 
     
     def carrying_capacity(self, fish) -> float:
-        length = self.n * self.D   # Based on worst case
-        #length = self.n * self.D + self.spacing * (self.n-1) # From reference paper for a row farm
-        #length = self.D # for each pen
-
-        #print('length',length)
+        #length = self.D   # Based on worst case
+        length = self.n * self.D + self.spacing * (self.n-1) # From reference paper for a row farm
```

Comment:
Consider providing abbreviated citation for the reference paper here 
```suggestion
        length = self.n * self.D + self.spacing * (self.n-1) # From reference paper for a row farm
```

Reply:
updated.